### DISCUSSION TITLE
Shopify Liquid support using Shopify's Theme Tools as a Guide?

### DISCUSSION THREAD

**Discussion by tylerhellner** [zed-industries/zed#9246] [Feature Requests]
I love Zed. It has completely replaced VSCode whenever I get to hack on TS, Svelte, or Astro files. You can imagine the bummer then when I have to edit a Shopify theme and have to boot up VSCode or Neovim to work on `.liquid` files. 

Is this the correct place to propose adding, at the very least, the Liquid LSP and Prettier plugin to Zed? 

[Shopify's Theme Tools Repository](https://github.com/Shopify/theme-tools) has all of the requisite parts to make this work. I was able to set up NeoVim myself with some basic Lua studying to run the LSP, get Syntax Highlighting and snippet completions, etc. Lua is one thing, but I have zero Rust knowledge. How feasible is it to add some of these Shopify packages to Zed? Is there any demand for it? Should I ask over on the Theme Tools repository if a Shopify developer can handle the work? 

Thank you for all the hard work on this editor. It's my favorite piece of software of the year so far. Apologies in advance if this is not the place for such a question.

**Comment by jeffreyguenther** [comment]
We're gonna do it with https://github.com/TheBeyondGroup/zed-shopify-liquid.

I know you know this because you've commented on a PR. For other readers, this is happening.
